src/components/molecules/paginator.rs

```diff
@@ -0,0 +1,67 @@
+use dioxus::prelude::*;
+use dioxus_std::{i18n::use_i18, translate};
+
+use crate::components::atoms::{dropdown::ElementSize, ArrowLeft, ArrowRight, Icon, IconButton};
+
+#[derive(PartialEq, Clone)]
+pub struct PaginatorValue {
+    value: u8,
```

Comment:
I have the impression that using `usize` here and in the props could save you some converting code and expects that are a bit ugly.